projects/packages/my-jetpack/_inc/components/product-cards-section/backup-card/index.jsx

```diff
@@ -128,12 +130,58 @@ const getTimeSinceLastRenewableEvent = lastRewindableEventTime => {
 const BackupCard = props => {
 	const { detail } = useProduct( productSlug );
 	const { status } = detail;
+	const { backup_failure: backupFailure } =
+		getMyJetpackWindowInitialState( 'redBubbleAlerts' ) || {};
+	const { status: lastBackupStatus } = backupFailure || {};
 	const hasBackups = status === PRODUCT_STATUSES.ACTIVE || status === PRODUCT_STATUSES.CAN_UPGRADE;
+	const noDescription = () => null;
 
-	return hasBackups ? (
-		<WithBackupsValueSection slug={ productSlug } { ...props } />
-	) : (
-		<ProductCard slug={ productSlug } { ...props } />
+	const { title: errorTitle, text: errorDescription } = useGetReadableFailedBackupReason() || {};
+
+	if ( hasBackups ) {
+		return <WithBackupsValueSection slug={ productSlug } { ...props } />;
+	}
+
+	return (
+		// eslint-disable-next-line react/jsx-no-bind
+		<ProductCard slug={ productSlug } Description={ noDescription } { ...props }>
+			{ status === PRODUCT_STATUSES.NEEDS_ATTENTION && backupFailure && (
+				<div className={ styles.backupErrorContainer }>
+					<div className={ styles.iconContainer }>
+						<Gridicon icon="notice" size={ 16 } className={ styles.iconError } />
+					</div>
+					<div className={ styles.contentContainer }>
+						<Text variant="body-small" className="value-section__heading">
+							{ __( 'The last backup attempt failed.', 'jetpack-my-jetpack' ) }
+							<InfoTooltip
+								tracksEventName={ 'protect_card_tooltip_open' }
+								tracksEventProps={ {
+									location: 'backup-error',
+									status: status,
+									backup_status: lastBackupStatus,
+									feature: 'jetpack-backup',
+									message: '', //errorDescription,
+								} }
+							>
+								<>
+									<h3>{ errorTitle }</h3>
+									<p>{ errorDescription }</p>
+									<p>
+										{ __(
+											'Check out our troubleshooting guide or contact your hosting provider to resolve the issue.',
+											'jetpack-my-jetpack'
+										) }
+									</p>
+								</>
+							</InfoTooltip>
+						</Text>
+						<Text variant="body-small" className={ styles.error_description }>
+							{ __( 'Check out our troubleshooting guide.', 'jetpack-my-jetpack' ) }
```

Comment:
This is not a strong opinion, but I think this text may be unnecessary since the CTA says `troubleshoot`. What do you think?

Reply:
Hmm, well that text _is_ in the Figma design, so I think I should maybe keep it there, at least for now.

![Screen Shot 2024-12-18 at 20 32 57](https://github.com/user-attachments/assets/996fb401-90b6-4adb-a17e-fd5bb081d686)

Figma design: v1IWiVmOAvI8yT8CCa4b1I-fi-4398_7392

Reply:
Is there a copy doc? I've been told the designs aren't always the be-all end-all when it comes to copy 😅